### ISSUE TITLE
feat(@vtmn/css): be able to choose the duration of display for snackbar (& toast)

### ISSUE THREAD

**Issue by Z26PGOBL** [Decathlon/vitamin-web#1460]
**Describe the bug**

We are implementing the snackbar in our application and for accessibility purpose our UI/UX team recommended that the snackbar should be displayed for 8 seconds.
We see that the duration is hardcoded in the `--vtmn-animation_overlay` custom property of [animations design-tokens](https://github.com/Decathlon/vitamin-web/blob/main/packages/sources/css/src/design-tokens/src/animations.css) so we manage to override it, but we think it would be good to be able to set it more "gracefully".

![Capture d’écran 2023-09-07 à 15 44 19](https://github.com/Decathlon/vitamin-web/assets/48095552/42562dde-a52d-4917-9d4c-cf7783e5de8c)

**Steps to reproduce**
    1. Go to https://decathlon.github.io/vitamin-web/@vtmn/showcase-css/?path=/docs/components-overlays-snackbar--demo
    2. Click on a demo button
    3. See the snackbar appear and disappear after about 5 seconds

**Expected behavior**
We should be able to set the duration time

**Comment by lauthieb** [comment]
Hello Pauline,

Normally, you can totally control the display duration of the VtmnSnackbar on your own. Here is an example I made with `@vtmn/react`: https://stackblitz.com/edit/vitejs-vite-rnupye?file=src%2FApp.jsx

**Comment by Z26PGOBL** [comment]
it works if you put a value smaller than 4,5 second, but if you put `8000` as timeout value the snackbar will still disappear after 4,5 second and not 8

**Comment by lauthieb** [comment]
Hi @Z26PGOBL, oh you're right, sorry!
That's a good catch. We take this feedback for our new implementation of Vitamin (currently inner source) & I've put this issue with "community" label. Feel free to contribute, I'll be there to review.
Thank you!
